src/reportSources/legacyFflogs/eventAdapter/deduplicateActorUpdates.ts

```diff
@@ -33,7 +33,7 @@ export class DeduplicateActorUpdateStep extends AdapterStep {
 			actor: next.actor,
 		} as const
 
-		type DeduplicatedFields = Omit<Events['actorUpdate'], keyof typeof baseFields>
+		type DeduplicatedFields = Omit<Events['actorUpdate'], keyof typeof baseFields | 'attributes'>
```

Comment:
It was mostly because denseObject was squawking due to adding the 'attributes' field to the type (as you noted it was going to in the comments on that function), but due to the ordering of the adapter steps there was never going to be an AU packet with the attributes property that would run through this adapter, and I was a little lazy.